conf/broker.conf

```diff
@@ -1360,6 +1360,10 @@ managedLedgerInfoCompressionThresholdInBytes=16384
 # corrupted at bookkeeper and managed-cursor is stuck at that ledger.
 autoSkipNonRecoverableData=false
 
+# Whether to allow brokers to forcefully load topics by skipping ledger failures to avoid topic unavailability and 
+# perform auto repairs of the topics.
+managedLedgerForceRecovery=false
+
```

Comment:
please also add this to `standalone.conf`. That is the default config file used for `pulsar standalone`.